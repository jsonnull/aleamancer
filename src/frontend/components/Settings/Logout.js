// @flow
import React from 'react'
import styled from 'styled-components'
import Button from '../Button'

type Props = {
  logout: Function
}

const Container = styled.div`
  margin-top: 2rem;
  display: flex;
`

const Logout = (props: Props) => {
  return (
    <Container>
      <Button red onClick={props.logout}>
        Logout
      </Button>
    </Container>
  )
}

export default Logout
