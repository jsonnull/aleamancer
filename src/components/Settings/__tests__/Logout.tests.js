// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import Logout from '../Logout.js'

describe('Settings Logout component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Logout logout={() => {}} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
