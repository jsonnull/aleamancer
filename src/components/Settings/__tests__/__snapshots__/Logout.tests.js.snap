// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Logout component renders correctly 1`] = `
<div
  className="sc-bwzfXH RlJjI"
>
  <button
    className="sc-bdVaJa gyRotp"
    onClick={[Function]}
  >
    Logout
  </button>
</div>
`;
